src/modules/alaca/src/components/dvn-pairs-meeting-card/dvn-pairs-meeting-card.tsx

```diff
@@ -0,0 +1,185 @@
+import { ReferenceField } from '@dvn/core/doldurma/src/fields/reference';
+import { ListFetchState } from '@dvn/core/models/list-fetch-state';
+import { TableConfig } from '@dvn/core/models/table-config';
+import { Component, h, Host, Prop, Event, EventEmitter, State } from '@stencil/core';
+import { AtamaApi } from 'src/modules/atama/src/api/atamaApi';
+import { MeetingGoalConfig } from '../../../../mentornity-js-client/models';
+
+
+
+@Component({
+  styleUrl: 'dvn-pairs-meeting-card.scss',
+  tag: 'dvn-pairs-meeting-card'
+})
+
+
+export class PairsMeetingCard {
+
+  @State() list: ListFetchState = ListFetchState.init();
+  @State() menteeType: any;
+
+  @Prop() meetingGoalConfig: MeetingGoalConfig;
+  @Prop() pairMeetingSummary: any;
+  @Prop() table: TableConfig;
+  @Prop() strings: any;
+  @Prop() userType: any;
+  @Prop() organisationConfig: any;
+  @Prop() params: any;
+
+  @Prop() apiAtama: AtamaApi;
+
+  @Event() action: EventEmitter;
+
+  fetchList() {
+    const { list, table } = this;
+    if (list.isPending || (list.error)) {
+      return;
+    }
+
+    this.list = list.beforeFetch();
+    const memberToShowTableName = (table.fields.find(
+      f => f.type === 'reference' && f.id === 'subscriber'
+    ) as ReferenceField)?.otherCollection;
+    this.menteeType = memberToShowTableName;
+
+    if (memberToShowTableName !== 'groups') {
+      this.apiAtama.getReceivedRequests(table.base, table.name)
+        .then(response => {
+          this.list = list.afterFetch(response, false);
+        }, error => {
+          this.list = list.afterFetchError(error);
+        });
+    }
+  }
+
+  componentWillLoad() {
+    this.fetchList();
+  }
+
+  render() {
+    const { pairMeetingSummary, organisationConfig, list, strings, userType, menteeType } = this;
+
+    // get pairs type
+    let pairsType;
+    if (userType === 'mentee') {
+      pairsType = 'mentor';
+    } else {
+      pairsType = menteeType === 'groups' ? 'groups' : 'mentee';
+    }
+
+    const emptyMessages: any[] = [];
+
+    if (userType === 'mentee') {
+      if (!pairMeetingSummary.length && organisationConfig.subscriptionConfig.mentorVisibilityStatus === 'showAvailableMentors') {
+        const message = (strings.notMatchedMessage || 'We couldnt see a {userType} matched with you at the moment.')
+          .replace(/{userType}/g, `${pairsType}`);
```

Comment:
Bu mesajların içinde rol parametreleri olmayacak. O yüzden replace kullanmana da gerek yok. Ayrıca default value da ekleme. Onlar diğer sayfalarda da kalkacak. Kısaca şu şekilde olacak.
```suggestion
        const message = strings.notMatchedMessage;
```